build.sh

```diff
@@ -394,7 +394,7 @@ isMSBuildOnNETCoreSupported()
                     __isMSBuildOnNETCoreSupported=1
                     ;;
                 "fedora.25-x64")
-                    __isMSBuildOnNETCoreSupported=1
+                    __isMSBuildOnNETCoreSupported=0
```

Comment:
Btw, there is a pending PR to fix that:
https://github.com/dotnet/coreclr/pull/12102